src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card__img,
+.card__product,
+.card__code,
+.card__rev,
+.card__rev-text,
+.card__price,
+.card__price-text,
+.card__cost,
+.card__button {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 33px 16px 16px;
+  width: 200px;
+
+  &__img {
```

Comment:
The `card__img` class doesn't have a width property set. The width should be set for consistency and to avoid content distortion.